packages/ms-spectrum/src/getMatchingScore.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const { xSum } = require('ml-spectra-processing');
+
+/**
+ * Filter the array of peaks
```

Comment:
You should add more documentation here. I still don't understand what this function does.